eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/json/EclairModels.scala

```diff
@@ -246,10 +237,22 @@ case class ChannelResult(
     state: ChannelState,
     feeBaseMsat: Option[MilliSatoshis],
     feeProportionalMillionths: Option[FeeProportionalMillionths],
-    data: JsObject)
+    data: JsObject) {
+  import JsonReaders._
+  def shortChannelId: ShortChannelId = (data \ "shortChannelId").validate[ShortChannelId].get
+}
 
 // ChannelResult ends here
 
+case class CreateInvoiceResult(
+  prefix: String,
+  timestamp: Long,
```

Comment:
can we add a better type to this?

Reply:
Maybe. Which one? All other Eclair models use Long for timestamps.

Reply:
I'm not picky, but just something that is more structured than `Long`. As you heard this morning during standup we have used `Long` in the past and been burned.

Reply:
I guess is the timestamp in milliseconds or seconds?